ports/gunroar/gunroar/gunroar.gptk

```diff
@@ -21,10 +21,10 @@ left_analog_down = down
 left_analog_left = left
 left_analog_right = right
 
-right_analog_up = \"
-right_analog_down = \"
-right_analog_left = \"
-right_analog_right = \"
+right_analog_up = i
+right_analog_down = k
+right_analog_left = j
+right_analog_right = l
 
 deadzone_mode = scaled_radial
```

Comment:
Can you remove these 4 deadzone lines, they make shooting diagonal nearly impossible, without them, it will use the defaults which work great.